SPECS/aws-nitro-enclaves-cli.spec

```diff
@@ -17,8 +17,8 @@
 
 Summary:    AWS Nitro Enclaves tools for managing enclaves
 Name:       aws-nitro-enclaves-cli
-Version:    1.0
-Release:    8%{?dist}
+Version:    1.0.9
```

Comment:
Should '9' be in the Release field?

Reply:
From offline discussions we decided to switch to a version format such as `1.0.9-0`.

Reply:
Ok, thanks for info.